iowrite.cpp

```diff
@@ -381,7 +381,7 @@ bool writeDataReducer(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>&
       }
    }
 
-   if( success ) {
+   if( dataReducer.getName(dataReducerIndex).find("fg_", 0) != 0 ) {
```

Comment:
So this check is true when the datareducer name does *not* start with `fg_`. This means that it now tries to go into this section for both DCCRG ionospheric reducers.

So I suggest that:
 * here we check if it *does* start with `fg_`, and then do the the code on current line 429 onwards
 * if that fails, check if it *does* start with `ig_`, and in that case perform the code on line 463 onwards
 * if both those fail, fallback to DCCRG data, and perform the lines 387-425

Reply:
Yes, this is the issue. I implemented this and do some testing now.